apps/docs/app/features/code-block/CodeBlock.tsx

```diff
@@ -16,7 +16,12 @@ export const CodeBlock = ({
   ...props
 }: CodeBlockProps) => {
   return (
-    <CodeBlockContainer {...props} code={code}>
+    <CodeBlockContainer
+      maxWidth={`calc(100vw - var(--spor-space-6))`}
```

Comment:
Dette er kjempeirriterende – man burde ikke trenge å sette denne maxWidth-en, men det kreves for å få overflow av kodeeksempler til å fungere som det skal.